intro-basics-2023-1/Assets/Scripts/Bullet.cs

```diff
@@ -25,7 +25,9 @@ private void OnCollisionEnter2D(Collision2D other)
     private void OnTriggerEnter2D(Collider2D other)
     {
         Debug.Log("Hit with " + other.name);
-        DestroyProjectile();
+        if(other.gameObject.layer != LayerMask.NameToLayer("DetectionPoint")){
```

Comment:
Buena idea! Aunque se puede quedar algo corta si luego necesitas verificar más de una Layer. O si el nombre de la layer cambia.
En esos casos se puede utilizar [Layer mask](https://answers.unity.com/questions/50279/check-if-layer-is-in-layermask.html) o actualizar la Matriz de colisión.